platform-controller/src/test/java/org/hobbit/controller/docker/ContainerStateObserverImplTest.java

```diff
@@ -16,6 +16,9 @@
  */
 package org.hobbit.controller.docker;
 
+import org.hobbit.controller.containers.docker.ContainerStateObserverImpl;
+import org.hobbit.controller.containers.docker.ContainerTerminationCallbackImpl;
+import org.hobbit.controller.containers.ContainerManager;
```

Comment:
When the tested class moves into a different package, the test classes should move, too.